Assets/Scripts/Combat/Minotaur Abilities/ActionRage.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ActionRage : Action
+{
+    override public int CONCENTRATION_COST { get { return 3; } }
+    override public int MIN_AP_COST { get { return 2; } }
+    override public TargetType TARGET_TYPE { get { return TargetType.SELF_ONLY; } }
+
+    private IEnumerator WaitForSpellCastAnimations(float fDuration)
```

Comment:
I would move this method to the Action class so that we can reuse it as needed (eg from ActionRegenerate)

Reply:
Done.